xapian-applications/omega/docs/omegascript.rst

```diff
@@ -875,6 +881,10 @@ $sub{A,B}
 Logical Operators:
 ==================
 
+For Logical Operators we allow trailing characters,non-number arguments and empty arguments.
+Reason is that logical operators compare their arguments based on whether they are empty or not.
+Ex: not returns true for empty string and false otherwise.
```

Comment:
I'm not really sure this explanation make things much clearer.  These operators only care about whether the value is an empty string or not an empty string.  So there aren't "trailing characters", and talking about "non-number arguments" risks suggesting that numeric arguments are in some way meaningful here.

Adding a note here is probably useful, but I think it just wants to explain that in this context OmegaScript treats an empty string as a "false" logical value and any non-empty string as "true".

Reply:
fixed in a fixup commit.